Tasks/ArchiveFilesV2/archivefiles.ts

```diff
@@ -86,9 +109,10 @@ function sevenZipArchive(archive: string, compression: string, files: string[])
         sevenZip.arg('-bb3');
     }
     sevenZip.arg(archive);
-    for (var i = 0; i < files.length; i++) {
-        sevenZip.arg(files[i]);
-    }
+
+    const fileList = createFileList(files);
+    sevenZip.arg('@' + fileList);
```

Comment:
Are we sure this works cross plat?

Reply:
I'm 99% sure. The command line is presented as xplat in the 7z docs.